website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -164,7 +162,83 @@ You can optionally also indicate that the column defined in `@sortBy` should be
 </Hds::Table>
 ```
 
-#### More Examples: internationalized column headers, overflow menu dropdown
+#### Custom sort callback
+
+To implement a custom sort callback on a columns, (1) add a custom function as the value for `sortingFunction` in the column hash, and (2) include a custom `onSort` action in your table invocation. This is useful for cases where the key might not be A-Z or 0-9 sortable by default, i.e., status, and you’re otherwise unable to influence the shape of the data in the model. Here is an example, code truncated for clarity:
+
+```handlebars{data-execute=false}
+<!-- app/templates/components/table.hbs -->
+<Hds::Table
+  @model={{this.model.data}}
+  @columns={{array
+      (hash 
+        key='status'
+        label='Status'
+        isSortable=true
+        sortingFunction=this.myCustomSortingFunction
+      )
+      (hash key='album' label='Album')
+      (hash key='year' label='Year')
+    }}
+  @onSort={{this.myCustomOnSort}}
+>
+  <!-- <:body> here -->
+</Hds::Table>
+```
+
+Here’s an example of what a custom sort function could look like. In this example, we are indicating that we want to sort on a status, which takes its order based on the position in the array:
+
+```javascript
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
```

Comment:
we don't have to port all the controller code here, let's focus on what is relevant
```suggestion
```